html/ui-dialoglist.js

```diff
@@ -1,31 +1,40 @@
-jQuery(document).on("click touch touchend", "span.myDialogs", function (event) {
-  jQuery("#dialogInfo").remove();
-  jQuery("#form").remove();
-  focusAtCanvas();
-
-  var oldCount = jQuery(this).attr("count");
-  var text = jQuery(this).children("h2").text();
-  var count = jQuery("span#dialog-btn-label").attr("count");
-
-  jQuery(this).attr("count", 0);
-  text = text.replace(` ${oldCount}`, "");
-  jQuery(this).children("h2").text(text);
-  text = jQuery("span#dialog-btn-label").text();
-
-  if (count - oldCount > 0) {
-    text = text.replace(` ${count}`, " " + (count - oldCount) + "");
-    jQuery("span#dialog-btn-label").text(text);
-    jQuery("span#dialog-btn-label").attr("count", count - oldCount);
-  } else {
-    text = text.replace(` ${count}`, "");
-    jQuery("span#dialog-btn-label").text(text);
-    jQuery("span#dialog-btn-label").attr("count", 0);
+var touchmoved;
+jQuery(document).on("click touch touchend", "span.myDialogs", function(event) {
+  if (touchmoved) {
+      return;
```

Comment:
I see that the indentation size is 2, but this line uses 4 spaces. Could you fix this?

Reply:
Set file indentation to 2 spaces consistently